cypress/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Formatting this file with prettier yields different results, just FYI :smile: I will most likely format it with prettier (it's set as my default formatter in VSCode), so I'll be seeing some annoyance in the future

Maybe we could all use prettier as a default formatter?

Reply:
We already use prettier (through ESLint), maybe not in the same way, maybe some configuration should be updated, but let's leave it for now and let's discuss that in the future.

Reply:
Personally I'm using auto-save, so since I'm not triggering save action explicitly I also don't format a file unless I need to, in other words unless it has style errors. I could switch to prettier, but since we are not linting JSONs there are no style errors being reported, so you shouldn't expect me reformat files that are already valid.

Reply:
> We already use prettier (through ESLint), maybe not in the same way

That's probably since JSON is not specified in the possible `extensions`, and that also probably requires some additional plugin :slightly_smiling_face: We could run `prettier` on files as a pre-commit hook. Also, prettier has a `--check` flag that makes it so it will fail when there are formatting differences.

Configuring it is out of scope for this PR, so I agree to tackle it in the future